services/requester/batch_tx_pool.go

```diff
@@ -88,19 +76,16 @@ func NewBatchTxPool(
 	eoaActivity := expirable.NewLRU[gethCommon.Address, time.Time](
 		eoaActivityCacheSize,
 		nil,
-		config.TxBatchInterval,
+		eoaActivityCacheTTL,
```

Comment:
why increase TTL from the batch interval to 10s?

Reply:
ok, let's make this configurable then. that way we don't have to play whack-a-mole with builds if this needs to be tuned later.